conans/test/functional/graph/test_graph_build_mode.py

```diff
@@ -0,0 +1,167 @@
+import pytest
+import operator
+from conans.test.assets.genconanfile import GenConanfile
+from conans.test.utils.tools import TestClient
+
+
+class fakeop:
```

Comment:
This is a bit cryptic, could you please explain a little bit?

Reply:
I would like to have a condition where I could return `in` or `not in` operator. This class is a trick to override these operators.

Reply:
A bit dark magic, I'd like to see the conditions explicitly in tests:
```python
if something:
    assert "some test" not in out
else:
    assert "some test" in out
```

Sometimes, because the ``not in`` are tricky and can easily pass for every minor formatting change to the output, they should look more like:

```python
if something:
    assert "ERROR" not in out # even if the full message is "ERROR: This didn't happen correctly"
else:
    assert "SUCCESS: This condition happened successfully" in out
```

Reply:
don't underestimate the power of the dark side! Indeed it's hard to read and worst to maintain. I'll revert to something simpler.